### PR TITLE
[release-0.18] Backport #1083 #1113

### PR DESCRIPTION
**Pull request by navidshaikh**
Backport PR for release-v0.18.4 with
- #1083
- #1113 
- Updated CHANGELOG

### CODE REVIEW

**Comment by navidshaikh** [comment]
@BbolroC : can you please add your consent by adding comment @googlebot I consent. here ?

File: hack/release.sh [hunk 1]
```diff
@@ -35,13 +35,15 @@ function build_release() {
...
-  ARTIFACTS_TO_PUBLISH="kn-darwin-amd64 kn-linux-amd64 kn-windows-amd64.exe kn-image-location.yaml"
-  if type sha256sum >/dev/null 2>&1; then
-    echo "🧮     Checksum:"
-    sha256sum ${ARTIFACTS_TO_PUBLISH}
-  fi
+  ARTIFACTS_TO_PUBLISH="kn-darwin-amd64 kn-linux-amd64 kn-linux-arm64 kn-windows-amd64.exe kn-linux-s390x kn-image-location.yaml"
+  sha256sum "${ARTIFACTS_TO_PUBLISH}" > checksums.txt
+  ARTIFACTS_TO_PUBLISH="${ARTIFACTS_TO_PUBLISH} checksums.txt"
+  echo "🧮     Checksum:"
+  cat checksums.txt
```
Comment (navidshaikh) [inline hack/release.sh:46]: /cc @rhuss 
PTAL

**Comment by BbolroC** [comment]
@googlebot I consent

**Review by rhuss — COMMENTED** [review]
/lgtm
/approve
/hold

File: hack/release.sh (hunk 1 above)
Comment (rhuss) [reply hack/release.sh:46]: looks good

**Comment by rhuss** [comment]
Not sure why @googlebot still doesn't like it.

If everything breaks, I found out that if you squash the PR manually then you can get over this. However this will loose the history of the commit

**Comment by rhuss** [comment]
@BbolroC could you please comment `@googlebot I consent.` (mind the trailing dot, that you forgot on your fist comment). Thx!

**Comment by BbolroC** [comment]
@googlebot I consent

**Comment by navidshaikh** [comment]
@googlebot I consent.

**Comment by navidshaikh** [comment]
@rhuss : looks like it commit has apparently another person as co-author whose github id no longer exists :thinking:

**Comment by rhuss** [comment]
@navidshaikh ok, in this case I suggest that you squash all commits locally on your branch and then force push it to this PR. That's the easiest way to overcome the CLA bot (which is stupid that he doesn't detect that these commits has been already consented in a different branch)

**Comment by navidshaikh** [comment]
there are two separate emails @gmail and @ibm.com of @BbolroC which seems to confuse the bot
```
commit 44d9f14f9f96f35a6daeebd276cf3d1d8ed3e918
Author: Hyounggyu Choi <bbolroc@gmail.com>
Date:   Tue Nov 24 20:51:35 2020 +0100

    Cross-compile the kn binary for linux/s390x (#1083)
    
    Co-authored-by: Hyounggyu Choi <Hyounggyu.Choi@ibm.com>
```

**Comment by navidshaikh** [comment]
@rhuss : I edited the commit and reset its author to keep the history, `➜  client git:(44d9f14f) git commit --amend --reset-author` 
bot is happy now :-)

**Comment by BbolroC** [comment]
I will keep this in mind and never make the confused again. Thank @navidshaikh for the care.

**Comment by navidshaikh** [comment]
/retest

        result_collector.go:75: ERROR: Error: Operation cannot be fulfilled on pingsources.sources.knative.dev "testpingsource0": the object has been modified; please apply your changes to the latest version and try again

**Pull request merged**